evm/src/cpu/kernel/asm/hash/blake2/compression.asm

```diff
@@ -22,9 +22,10 @@ compression_loop:
     POP
     // stack: cur_block, retdest
     PUSH 0
+    PUSH 1
```

Comment:
Shouldn't this `PUSH 1` be before the `PUSH 0`?

Reply:
Oops good catch, also updated the comment above `SUB`